### PR TITLE
Add checklists processor

### LINKED ISSUE #12: Checklists
## Description
Common lists are defined in the global css, but there are some cases where we'd want to change the marker to a check, instead of the default one.

## How it works
For the lists that have the class `check-marker-list` we should change the marker to this:
```
list-style-image: url(https://wp.frontity.org/wp-content/uploads/2020/02/check-circle-marker.svg)
```

## Where to check
* Inside Invision Homepage -> Why WP & Why React.
* Inside common-styles page -> Check marker list

### PR DESCRIPTION
**Pull request by iamuchejude**
Fixes #12

### CODE REVIEW

**Comment by frontibotito** [comment]
Deploy preview for _website_ ready!

Built with commit 9e880059896fdcec421a37c38a93a6b729bb432f

https://frontity-org-g4g3usxso.now.sh

**Comment by iamuchejude** [comment]
@SantosGuillamot Checklist Icons is now centered.

**Comment by iamuchejude** [comment]
Oh! I'm supposed to delete that. I'm gonna fix it now.

File: packages/frontity-org-theme/src/components/styles/global-styles.tsx [hunk 1]
```diff
@@ -114,6 +114,16 @@ const elementBase = (colors: FrontityOrg["state"]["theme"]["colors"]) => css`
   }
   ul {
     list-style-type: "→ ";
+
+    & > li {
+      margin: 1.1rem 0;
```
Comment (SantosGuillamot) [inline packages/frontity-org-theme/src/components/styles/global-styles.tsx]: This margin doesn't match the design. According to it, the margin would be 16px, which equals `margin: 1.125rem 0`.

File: packages/frontity-org-theme/src/components/styles/global-styles.tsx [hunk 2]
```diff
@@ -114,6 +114,16 @@ const elementBase = (colors: FrontityOrg["state"]["theme"]["colors"]) => css`
...
+      line-height: 1.3;
```
Comment (SantosGuillamot) [inline packages/frontity-org-theme/src/components/styles/global-styles.tsx]: According to the design, the line-height is 24px, which equals `line-height: 1.5rem`.

File: packages/frontity-org-theme/src/components/styles/global-styles.tsx [hunk 3]
```diff
@@ -114,6 +114,16 @@ const elementBase = (colors: FrontityOrg["state"]["theme"]["colors"]) => css`
...
+
+      &::before {
+        line-height: 0;
+        margin-right: 0.5rem;
```
Comment (SantosGuillamot) [inline packages/frontity-org-theme/src/components/styles/global-styles.tsx]: If you change the line-height to `1.5rem` as suggested before, it's higher than the height of the icon, so you'll need to add `auto` margin to the top and the bottom to center it vertically. I think you need to change the `margin-right` to `margin: auto 0.5rem auto 0` right?

Comment (iamuchejude) [reply packages/frontity-org-theme/src/components/styles/global-styles.tsx]: Yeah, that works.

**Comment by iamuchejude** [comment]
@luisherranz @SantosGuillamot This PR awaits your reviews.

File: packages/frontity-org-theme/src/processors/checklists.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import { css } from "frontity";
+
+const checklistsProcessor = {
```
Comment (luisherranz) [inline packages/frontity-org-theme/src/processors/checklists.ts]: Types for the processor are missing.

Comment (iamuchejude) [reply packages/frontity-org-theme/src/processors/checklists.ts]: I've fixed that.

**Pull request merged**